feature-libs/order/components/order-details/order-detail-items/consignment-tracking/augmented-features-config.model.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import '@spartacus/core';
+
+declare module '@spartacus/core' {
+  interface FeaturesConfig {
+    /**
+     * Enables displaying consignment tracking info in the order details view
+     */
+    consignmentTracking?: boolean;
+  }
```

Comment:
It's a real feature toggle, which is enabled by default since v1.2, not a breaking changes flag. To be discussed with PO (Steve), whether we should deprecate and eventually remove this toggle.